lib/commands/debug/helper.js

```diff
@@ -0,0 +1,143 @@
+const { execSync } = require('child_process');
+const path = require('path');
+const fs = require('fs');
+const nodemon = require('nodemon');
+const CliError = require('@src/exceptions/cli-error');
+
+module.exports = {
+    getSkillInvocationInfo,
+    initiateDebugging
+};
+
+function getPythonDebugAdapterPath() {
+    let localDebuggerPath;
```

Comment:
You can probably rewrite to more functional code
```
const sitePkgLocationsStr = execSync('python3 -c "import site; import json; print(json.dumps(site.getsitepackages()))"').toString();
const sitePkgLocations = JSON.parse(sitePkgLocationsStr);
const localDebuggerPath = sitePkgLocations
    .map(location => path.join(location, 'ask_sdk_local_debug', 'local_debugger_invoker.py'))
    .find(location => fs.existsSync(location));
if (!localDebuggerPath) {
    throw new CliError('Some error message');
}
return localDebuggerPath;
```

Reply:
Thanks!